.pipelines/azure_pipeline_dev.yaml

```diff
@@ -0,0 +1,76 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- ci_dev
+
+pool:
+  name: Azure-Pipelines-CI-Test-EO
+
+variables:
+  armServiceConnectionName: 'ci-1es-acr-connection'
+  subscription: '9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb'
+  containerRegistry: 'containerinsightsprod'
+  repoImageName: '${{ variables.containerRegistry }}.azurecr.io/public/azuremonitor/containerinsights/cidev'
+
+steps:
+- bash: |
+    commit=$(git rev-parse --short HEAD)
+    echo "##vso[task.setvariable variable=commit;]$commit"
+
+    datetime=$(date +'%Y%m%d%s')
+    echo "##vso[task.setvariable variable=datetime;]$datetime"
+
+    cd deployment/multiarch-agent-deployment/ServiceGroupRoot/Scripts
+    tar -czvf ../artifacts.tar.gz pushAgentToAcr.sh
```

Comment:
I believe we will also want to tar the artifacts for the chart release here as well. reference:
https://github.com/microsoft/Docker-Provider/blob/01e8178925b3fa284952f49014e42ec3e5abb1bd/.pipelines/build-linux.sh#L25

We may not need this yet but commenting to flag it

Reply:
Currently the pipelines + builds for multiarch and single plat are different. Once we merge them then the above changes will be added